apps/heft/src/schemas/heft.schema.json

```diff
@@ -131,12 +129,12 @@
 
                         "flatten": {
                           "type": "boolean",
-                          "description": "Copy only the file and discard the relative path from the source folder. This defaults to false."
+                          "description": "Normally, when files are selected under a child folder, a corresponding folder will be created in the destination folder.  Specify flatten=true to discard the source path and copy all matching files to the same folder.  If two files have the same name an error will be reported. The default value is false."
                         },
 
                         "hardlink": {
                           "type": "boolean",
-                          "description": "Hardlink files instead of copying. This defaults to false."
+                          "description": "If true, filesystem hard links will be created instead of copying the file.  Depending on the operating system, this may be faster. (But note that it may cause unexpected behavior if a tool modifies the link.)  The default value is false."
```

Comment:
I think it's not grammatically correct to have a period inside of the brackets, or to have brackets be their own sentence... but it's been a while since English lessons :)

Reply:
Seems fine: https://www.thepunctuationguide.com/parentheses.html